arch/arm_cm/export/arch/execution.hpp

```diff
@@ -101,6 +127,25 @@ uint32_t events();
 
 #endif // USE_SYSTMR
 
+//! \brief Performs a dummy busy wait for specified amount of milliseconds.
+//! \param ms number of milliseconds to wait.
+//!
+//! This function is useful for a short delays.
+//!
+//! \return None.
+static inline void arch_spin_wait(uint32_t ms)
```

Comment:
I propose to add on-device test for that (I think it can be tested both in STM32 and TM4C platform)